system/lib/dlmalloc.c

```diff
@@ -872,7 +874,9 @@ struct mallinfo mallinfo(void) __attribute__((weak, alias("dlmallinfo")));
 #endif
 int mallopt(int, int) __attribute__((weak, alias("dlmallopt")));
 int malloc_trim(size_t) __attribute__((weak, alias("dlmalloc_trim")));
+#if !NO_MALLOC_STATS
 void malloc_stats(void) __attribute__((weak, alias("dlmalloc_stats")));
+#endif
```

Comment:
Is this needed for something?

Reply:
`malloc_stats`? it's a GNU extension AFAIK, that maybe some people use http://man7.org/linux/man-pages/man3/malloc_stats.3.html

Reply:
Works for me

Reply:
Oh, it fails to compile without the ifdef. If `NO_MALLOC_STATS` is set, it isn't defined, so we can't make a weak alias of it.